docs/_config.yml

```diff
@@ -4,7 +4,7 @@ title: systemd
 baseurl: "" # the subpath of your site, e.g. /blog/
 url: "https://systemd.io" # the base hostname & protocol for your site
 
-permalink: /:title/
+permalink: /:title
```

Comment:
Please explain what is the effect of this change (in the commit message). We care about URL stability.

Reply:
Thanks!